plugins/aws/aws.plugin.zsh

```diff
@@ -143,20 +169,29 @@ function aws_change_access_key() {
   AWS_PAGER="" aws iam list-access-keys
 }
 
+function aws_regions() {
```

Comment:
Is there a way to query regions from AWS instead of hard-coding them?

Reply:
modified!